### PR TITLE
Support for stdin

### PR DESCRIPTION
**Pull request by marcglobality**
Adding support for input from stdin. Motivation here is to add (better) support for VScode, since the formatters there use stdin and stdout to format the open file. See README.md on how to config VScode

Note: many of the changes are dummy and due to new versions of black & mypy + trailing spaces

### CODE REVIEW

**Review by tpietruszka — COMMENTED** [review]
This is a great initiative!

Left some comment - but I think we could have a slightly different interface: more consistent with `black` and enabling all functionality, like `--diff` and `--check` with STDIN, instead of having a separate code path.

Not sure if ☝️ makes sense, I will make a PR to this branch.


Also, VS Code instructions didn't work for me (or I applied them wrong). Will experiment more once we settle on the exact interface.

**Comment by tpietruszka** [comment]
My suggestions are in #32

**Comment by marcglobality** [comment]
> My suggestions are in #32

Thanks @tpietruszka ! Merged into this and updated the README. This is now ready

**Review by salmanmashayekh — APPROVED** [review]
I don't have much context here Marc. Plz address Tomasz's comments, as he is the one with much better understanding than me. Approving conditional on that.

File: README.md [hunk 1]
```diff
@@ -124,40 +133,95 @@ Then, go to the extension preferences, and add:
 ```
 
 Notes:
- - The last step above translates into user settings saved in 
+ - The last step above translates into user settings saved in
 `~/.jupyter/lab/user-settings/@ryantam626/`.
```
[the remark this answers is not in the document]
Comment (marcglobality) [reply README.md:137]: this is the user who created the plugin https://github.com/ryantam626/jupyterlab_code_formatter for JupyterLab. Just mentioning that when configuring his extension, this new thing will appear

File: globality_black/comprehensions.py [hunk 2]
```diff
@@ -96,7 +96,6 @@ def _reformat_comprehension(comp_for: PythonNode):
 
 
 def set_prefix(element: PythonNode, prefix: str):
-
```
[the remark this answers is not in the document]
Comment (marcglobality) [reply globality_black/comprehensions.py:99]: Has black changed? yes

File: README.md [hunk 3]
```diff
@@ -41,16 +50,16 @@ Installation
...
 -----
 
-There are two ways to use `globality-black`, via CLI, or importing the helpers in the library. 
+There are two ways to use `globality-black`, via CLI, or importing the helpers in the library.
 Next, we show some typical use cases:
 
 ### CLI
 
-Please see command line arguments running `globality-black --help`. 
+Please see command line arguments running `globality-black --help`.
```
[the remark this answers is not in the document]
Comment (marcglobality) [reply README.md:58]: added

File: globality_black/cli.py [hunk 4]
```diff
@@ -67,17 +94,17 @@ def main(path, check, diff, verbose):
         paths = [path]
 
     reformatted_count, failed_count = 0, 0
-    process_path_with_check = partial(process_path, check_only_mode=check, diff_mode=diff)
+    process_path_with_check = partial(process_src, check_only_mode=check, diff_mode=diff)
```
[the remark this answers is not in the document]
Comment (marcglobality) [reply globality_black/cli.py:77]: done

**Pull request merged**